docs/traffic-policy/actions/deny/tls/examples/tls-basic-example.mdx

```diff
@@ -0,0 +1,35 @@
+import ConfigExample from "/src/components/ConfigExample.tsx";
+
+### Deny all traffic
+
+The following [Traffic Policy](/docs/tls/traffic-policy/index.mdx) 
+configuration will deny all inbound traffic on your endpoint.
+
+#### Example Traffic Policy Document
+
+<ConfigExample
+  snippetText={null}
+  showLineNumbers={true}
+  yamlMetastring="{4-10}"
+  jsonMetastring="{5-7}"
+  config={{
+    inbound: [{
+      actions: [
+        {
+          type: "deny"
+        },
+      ],
+    }]
+  }}
+/>
+
+#### Example Request
+
+```shell
+$ curl 12e461b71f86488a.ngrok.app:443
```

Comment:
nit: not sure if this gives us signal on tls + deny (eg i think without deny and tls terminated at edge and no online agent you get the same), but don't have a better example yet

Reply:
When I tried it locally I got a Signal